### PR TITLE
add simple-peer v5.11.5

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/feross/simple-peer
**Watch 26
  Star 458
 Fork 55**
@Piicksarn could u help me check this PR for #6623 ? thank you!

- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity 
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/simple-peer/5.11.6/simplepeer.min.js [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (PeterDaveHello) [inline ajax/libs/simple-peer/5.11.6/simplepeer.min.js]: Why to add a empty file here? Don't you need you check your own commit?

**Comment by maruilian11** [comment]
@PeterDaveHello sorry for checking carelessly, i will open an issue to discuss with the author.

**Comment by maruilian11** [comment]
https://github.com/feross/simple-peer/issues/52
issue already be opened.

**Comment by maruilian11** [comment]
@PeterDaveHello i have changed the version to v5.11.5.

**Comment by PeterDaveHello** [comment]
Thanks.

**Pull request merged**